### ISSUE TITLE
spotify.getData is not a function

### ISSUE THREAD

**Issue by kaname-png** [microlinkhq/spotify-url-info#85]
**Describe the bug**
I get this error in this new update, it did not happen in the previous one and the code that this package uses has not been changed.

**Spotify URL(s)**
https://open.spotify.com/track/4tRluQ3OVRGDE9CMTALg3t?si=d7db95dd2a694aa7

**Your Code**
```ts
import * as spotify from 'spotify-url-info';
await spotify.getData(url);
```

**Error Message**
```
PS E:\Proyectos\Personal\Naeko\naeko> yarn start
SyntaxError: The requested module 'spotify-url-info' does not provide an export named 'getData'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:127:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:193:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:337:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)

Node.js v17.5.0
```

**Comment by Kikobeats** [comment]
Hello,

Since v3, you have to provide your fetch agent.

See https://github.com/microlinkhq/spotify-url-info/tree/v3.0.0 notes 🙂

**Comment by kaname-png** [comment]
@Kikobeats Oh, I see, but the types were not updated. 😿

**Comment by Kikobeats** [comment]
oh, sorry for that. I totally forgot about that.

I'm not sure how to write the new types. Can you make a PR?

**Comment by kaname-png** [comment]
@Kikobeats sure.
https://github.com/microlinkhq/spotify-url-info/pull/86

**Comment by Kikobeats** [comment]
Released at [3.0.4](https://github.com/microlinkhq/spotify-url-info/commit/fa03e2884797d64fcaafd8bc401d27954c9ef1a6) 🎉
